web/src/gql/queries/workspace.ts

```diff
@@ -0,0 +1,156 @@
+import { gql } from "@apollo/client";
+
+export const WORKSPACES = gql`
+  fragment Workspace on Workspace {
+    id
+    name
+    members {
+      user {
+        id
+        name
+        email
+      }
+      userId
+      role
+    }
+    personal
+  }
```

Comment:
I'm pretty sure you can pull this out as its own variable and use it inside UPDATE_WORKSPACE and ADD_MEMBER_TO_WORKSPACE as well!
If you want you can also create a fragments folder and put it there (like we do in Re:Earth)? But at least move it out since you use this structure in update and add members!